astropy/utils/tests/test_data.py

```diff
@@ -60,6 +60,7 @@
 from astropy.utils.exceptions import AstropyWarning
 
 CI = os.environ.get("CI", False) == "true"
```

Comment:
Now that I think about it... Is this a bug? 😱 
```suggestion
CI = os.environ.get("CI", "false") == "true"
```

Reply:
That would be cleaner for sure, though should not change the outcome. 😺

Reply:
No, but my head hurts less. 😆